thunder/clang/__init__.py

```diff
@@ -19,7 +19,7 @@
 import thunder.core.dtypes as dtypes
 from thunder.core import utils
 import thunder.core.prims as prims
-from thunder.core.proxies import TensorProxy, pyval, pytype, proxy, AnyProxy, Proxy
+from thunder.core.proxies import IntegerProxy, NumberProxy, TensorProxy, pyval, pytype, proxy, AnyProxy, Proxy
```

Comment:
Note to myself. NumberProxy needs to be added on signature here. I'll do a clean up.